lib/resultsprocessor.rb

```diff
@@ -90,7 +92,7 @@ def parse_regression_line(line)
   def process_regression_results(stdout, stderr, result)
     results = []
 
-    stdout.split("\n").each { |line|
+    stdout.encode('UTF-8', :invalid => :replace).split("\n").each {|line|
```

Comment:
Adding these encoding function calls allows the CI to parse all the output reliably.